schema.sql

```diff
@@ -9,3 +9,29 @@ CREATE TABLE animals (
   weight_kg DOUBLE PRECISION NOT NULL,
 	species VARCHAR(50) NOT NULL
 );
+
+/* wednesday */
+/* New table for owners */
+CREATE TABLE owners (
+	id BIGSERIAL NOT NULL PRIMARY KEY,
+	full_name VARCHAR(100) NOT NULL,
+	age INTEGER NOT NULL
+);
+
+/*New table for species */
+CREATE TABLE species (
+	id BIGSERIAL NOT NULL PRIMARY KEY,
+	name VARCHAR(100) NOT NULL
+);
+
+/*Modify table */
+CREATE TABLE animals (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name varchar(100) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  escape_attempts INT NOT NULL,
+  neutered BOOLEAN NOT NULL,
+  weight_kg DOUBLE PRECISION NOT NULL
+  species_id INT NULL REFERENCES species(id) ON DELETE CASCADE,
+  owner_id INT NULL REFERENCES owners(id) ON DELETE CASCADE,
+);
```

Comment:
- [ ] As per the requirements, modify the table and not recreate it with updated columns. I see you did it right according to your screenshot but kindly replace this with the commands you used as shown in your screenshot as the table is already being created on lines 3-11 of your `schema.sql`. it should look like the example code below.

```sql
   ALTER TABLE REMOVE species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```